database/DevHome.Database/Migrations/DevHomeDatabaseContextModelSnapshot.cs

```diff
@@ -53,6 +53,11 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     .HasColumnType("TEXT")
                     .HasDefaultValue(string.Empty);
 
+                b.Property<Guid?>("SourceControlClassId")
+                    .ValueGeneratedOnAdd()
+                    .HasColumnType("TEXT")
+                    .HasDefaultValue(new Guid("00000000-0000-0000-0000-000000000000"));
```

Comment:
Can we use one of these instead (here and above)?
```
new Guid()
default(Guid)
Guid.Empty
```
https://stackoverflow.com/questions/30977651/what-is-the-default-value-for-guid

Reply:
Done.